samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -358,17 +365,26 @@ void main(void)
 
 		create_sync();
 
+		sync_wait = true;
+		sync_terminated = false;
```

Comment:
I am not sure, but maybe it is better to set these two flags before `create_sync` function in order to prevent any race condition. (for example if `term_cb` is executed exactly after when `sync_wait = true` is set but `sync_terminated = false` is not)

Reply:
Changed as suggested. It is better place to have the flags set before `create_sync`.